fuels-core/src/code_gen/functions_gen.rs

```diff
@@ -147,6 +147,10 @@ fn expand_function_arguments(
         // Note that _any_ significant change in the way the JSON ABI is generated
         // could affect this function expansion.
         // TokenStream representing the name of the argument
+        if param.type_field == "()" || param.type_field == "" {
```

Comment:
Is there a reason to have both these cases? Presumably the compile will only use one, so only one should be used here.

Reply:
I will remove the `""` if you think that's confusing